website/docs/guides/ttl-indexes/1-create-ttl-index.request.js

```diff
@@ -1,4 +1,5 @@
 db.runCommand({
   createIndexes: 'books',
-  indexes: [{ key: { 'reservation.date': 1 }, name: 'reservation_ttl', expireAfterSeconds: 60 }]
+  indexes: [{ key: { 'reservation.date': 1 }, name: 'reservation_ttl', expireAfterSeconds: 60 }],
+  $db: 'db'
```

Comment:
I saw this after updating my PR as well -- do we want to include it in the database within the requests?

Reply:
https://github.com/OpenDocDB/cts/issues/44